docs/reference/esql/functions/parameters/date_trunc.asciidoc

```diff
@@ -0,0 +1,9 @@
+// This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+
+*Parameters*
+
+`interval`::
+Interval; expressed using the timespan literal syntax.
```

Comment:
If someone tries `date_trunc(1 year + 1 day, ...)`, they get a 400 and a IAE: `{"type":"illegal_argument_exception","reason":"Time interval is not supported"}`. The docs here are correct, but our error message is misleading because `1year + 1 day` is a correct time interval.

Reply:
Probably `Time interval with multiple periods is not supported` would be more clear